cmd/minishift/cmd/image/export.go

```diff
@@ -69,7 +69,7 @@ func exportImage(cmd *cobra.Command, args []string) {
 	}
 	err = handler.ExportImages(imageCacheConfig)
 	if err != nil {
-		atexit.ExitWithMessage(1, fmt.Sprintf("Export of container images failed: %v", err))
+		atexit.ExitWithMessage(1, fmt.Sprintf("Failed to export the container image: %v", err))
```

Comment:
sorry if my comment wasn't clear, what I mean is that we should clarify how many images we are referring to, whether the users can select certain images, only one image, or all images? these are the types of questions that are most likely to be asked by users.